apps/common/behaviours/casework-submission.js

```diff
@@ -6,10 +6,9 @@ const client = new StatsD();
 
 const Compose = func => superclass => class extends superclass {
   prepare() {
-    if (typeof func === 'function') {
```

Comment:
func is always a function as the 4 journeys this Compose function is called in the submission process, a function which builds the payload to send to iCasework is called here